web_widget_plotly_chart/__manifest__.py

```diff
@@ -0,0 +1,17 @@
+{
+    "name": "Web Widget Plotly",
+    "summary": """Allow to draw plotly charts.""",
+    "author": "LevelPrime srl, " "Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/web",
+    "category": "Web",
+    "version": "14.0.1.0.0",
+    "depends": ["web"],
+    "data": [
+        "views/web_widget_plotly_chart.xml",
+    ],
+    "external_dependencies": {
+        "python": ["plotly==5.4.0"],
+    },
+    "auto_install": False,
```

Comment:
This is the default value

Reply:
@tarteo I've stick the python plotly version to the js one provided on the lib in order to avoid clashes.

Reply:
That's OK, it's about `auto_install`, it's by default `False`. But it's just a non-blocking comment :smile:

Reply:
@tarteo Doesn't matter I did it :)!